src/main/java/com/piekoszek/nowaksharedrent/hash/HashService.java

```diff
@@ -0,0 +1,7 @@
+package com.piekoszek.nowaksharedrent.hash;
+
+public interface HashService {
+
+    String encryptString (String input);
```

Comment:
just encrypt, we see that String is in parameter, maybe text instead of input, but here doesn't matter 2 much.